pkg/loader/util.go

```diff
@@ -171,6 +200,37 @@ func CreateDB(user string, password string, host string, port int, tls *tls.Conf
 	return CreateDBWithSQLMode(user, password, host, port, tls, nil, nil, time.Minute)
 }
 
+//CreateOracleDB create Oracle DB connection and return it
+func CreateOracleDB(user string, password string, host string, port int, serviceName, connectString string) (db *gosql.DB, err error) {
+	loc, err := time.LoadLocation("Local")
+	if err != nil {
+		return nil, err
+	}
+	pConnectString := ""
+	if connectString == "" {
+		if serviceName == "" {
+			return nil, errors.New("service-name should not be empty")
+		}
+		pConnectString = fmt.Sprintf("%s:%d/%s?connect_timeout=2", host, port, serviceName)
```

Comment:
Why `connect_timeout=2`?

Reply:
timeout time of connecting to db suggested by dba